packages/cli/src/commands/teams/switch.ts

```diff
@@ -71,13 +71,17 @@ export default async function main(client: Client, desiredSlug?: string) {
     }
 
     const choices = [
-      { separator: 'Personal Account' },
-      {
-        name: `${user.name || user.email} (${user.username})${suffix}`,
-        value: user.username,
-        short: user.username,
-        selected: personalScopeSelected,
-      },
+      ...(user.version === 'northstar'
+        ? []
+        : [
+            { separator: 'Personal Account' },
+            {
+              name: `${user.name || user.email} (${user.username})${suffix}`,
+              value: user.username,
+              short: user.username,
+              selected: personalScopeSelected,
+            },
+          ]),
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏extract into variable, or something

I find this complex set of expressions hard to read. Can we extract some named values or otherwise make this clearer?

Reply:
Fixed by 761ff070f